File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-03/plagiarized/L2/06/cabang.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function calculateBMI(): void {
  rl.question('Enter weight in pounds: ', (weightStr) => {
    const weight = parseFloat(weightStr);
    rl.question('Enter feet: ', (feetStr) => {
      const feet = parseFloat(feetStr);
      rl.question('Enter inches: ', (inchesStr) => {
        const inches = parseFloat(inchesStr);
        const height = feet * 12 + inches;
        const totalBMI = weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254));
        console.log(`BMI is ${totalBMI}`);
        if (totalBMI < 18.5) {
          console.log('Underweight');
        } else if (totalBMI < 25) {
          console.log('Normal');
        } else if (totalBMI < 30) {
          console.log('Overweight');
        } else {
          console.log('Obese');
        }
        rl.close();
      });
    });
  });
}

calculateBMI();

File: datasets/IR-Plag/convertai-original-java-rest-python/case-03/plagiarized/L4/06/cabang.py
import math

def main():
    weight = float(input("Enter weight in pounds: "))
    feet = float(input("Enter feet: "))
    inches = float(input("Enter inches: "))
    height = feet * 12 + inches
    total_bmi = calculate_bmi(weight, height)
    print("BMI is", total_bmi)
    display_result(total_bmi)

def calculate_bmi(weight, height):
    return weight * 0.45359237 / ((height * 0.0254) * (height * 0.0254))

def display_result(total_bmi):
    if total_bmi < 18.5:
        print("Underweight")
    elif total_bmi < 25:
        print("Normal")
    elif total_bmi < 30:
        print("Overweight")
    else:
        print("Obese")

if __name__ == "__main__":
    main()

